mantidproject/mantid
# Crystallography convention bug fixed

**Description of work.**

**Report to:** [user name]/[nobody]. 

**To test:**
Do following script with/without Crystallography convention in preferences.  Open TOPAZ_3132_md in sliceviewer and load peaks workspace.  Check that peaks overlay higher intensities in data.
````python
Load(Filename='~/mantid/release/ExternalData/Testing/Data/SystemTest/TOPAZ_3132_event.nxs', OutputWorkspace='TOPAZ_3132_event')
FilterBadPulses(InputWorkspace='TOPAZ_3132_event', OutputWorkspace='TOPAZ_3132_event', LowerCutoff=25)
ConvertToMD(InputWorkspace='TOPAZ_3132_event', QDimensions='Q3D', dEAnalysisMode='Elastic', Q3DFrames='Q_sample', LorentzCorrection=True, OutputWorkspace='TOPAZ_3132_md', MinValues='-25,-25,-25', MaxValues='25,25,25', SplitInto='2', SplitThreshold=50, MaxRecursionDepth=13, MinRecursionDepth=7)
FindPeaksMD(InputWorkspace='TOPAZ_3132_md', PeakDistanceThreshold=0.37680000000000002, MaxPeaks=10, DensityThresholdFactor=100, OutputWorkspace='TOPAZ_3132_peaks')

````

Fixes #23346 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
